test/WebJobs.Script.Tests/WebScriptHostRequestManagerTests.cs

```diff
@@ -1,114 +0,0 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the MIT License. See License.txt in the project root for license information.
-
-#if WEBREQUESTMANAGER
-using System.Collections.ObjectModel;
-using System.Diagnostics;
-using System.Linq;
-using System.Net;
-using System.Net.Http;
-using System.Threading;
-using System.Threading.Tasks;
-using Microsoft.Azure.WebJobs.Extensions.Http;
-using Microsoft.Azure.WebJobs.Script.Config;
-using Microsoft.Azure.WebJobs.Script.Description;
-using Microsoft.Azure.WebJobs.Script.Diagnostics;
-using Microsoft.Azure.WebJobs.Script.WebHost;
-using Moq;
-using Xunit;
-
-namespace Microsoft.Azure.WebJobs.Script.Tests
-{
-    public class WebScriptHostRequestManagerTests
-    {
-        private readonly WebScriptHostRequestManager _requestManager;
-        private readonly Mock<IMetricsLogger> _metricsLogger;
-        private readonly Mock<HostPerformanceManager> _performanceManager;
-        private readonly FunctionDescriptor _functionDescriptor;
-        private readonly HttpExtensionConfiguration _httpConfig;
-        private readonly TestTraceWriter _traceWriter;
-
-        public WebScriptHostRequestManagerTests()
-        {
-            _metricsLogger = new Mock<IMetricsLogger>(MockBehavior.Strict);
-            _performanceManager = new Mock<HostPerformanceManager>(MockBehavior.Strict, new object[] { new ScriptSettingsManager(), new HostHealthMonitorConfiguration() });
-            _httpConfig = new HttpExtensionConfiguration();
-            _traceWriter = new TestTraceWriter(TraceLevel.Verbose);
-            _requestManager = new WebScriptHostRequestManager(_httpConfig, _performanceManager.Object, _metricsLogger.Object, _traceWriter, 1);
-            _functionDescriptor = new FunctionDescriptor("Test", null, null, new Collection<ParameterDescriptor>(), null, null, null);
-        }
-
-        [Fact]
-        public async Task ProcessRequestAsync_PerformanceThrottle_ReturnsExpectedResult()
```

Comment:
This test was moved to the test class above and renamed Invoke_UnderLoad_RequestsAreRejected